### PR TITLE
SOF-7376: SOC tutorial

### PR DESCRIPTION
**Pull request by pranabdas**
Preview: https://deploy-preview-273--mat3ra-documentation.netlify.app/tutorials/dft/electronic/spin-orbit-coupling-qe/

### CODE REVIEW

**Comment by pranabdas** [comment]
For longer voice synthesis, we may need to split the content into smaller json files:
```
google.api_core.exceptions.InvalidArgument: 400 Either `input.text` or `input.ssml` is 
longer than the limit of 5000 bytes. This limit is different from quotas. To fix, reduce the 
byte length of the characters in this request, or consider using the Long Audio API: 
https://cloud.google.com/text-to-speech/docs/create-audio-text-long-audio-synthesis.
```

File: mkdocs.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ extra_javascript:
 copyright: <a target="_blank" href="https://mat3ra.com">Exabyte Inc</a>. All rights reserved. | <a target="_blank" href="https://platform.mat3ra.com">Back to platform</a>
 
 extra:
-    version: "2024.3.14"
+    version: "2024.5.16"
```
Comment (pranabdas) [inline mkdocs.yml:30]: Remember to update the latest platform version before merging.

Comment (timurbazhirov) [reply mkdocs.yml:30]: The latest version is 2024.8.22

File: lang/en/docs/tutorials/dft/electronic/soc-qe.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# How to incorporate spin-orbit coupling effect in Quantum ESPRESSO
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/dft/electronic/soc-qe.md:1]: soc -> spin-orbit-coupling in all files, please

File: lang/en/docs/tutorials/dft/electronic/soc-qe.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+limit for a given calculation, queue, number of nodes, and number of processor
+cores per node.
+
+![Compute parameters](/images/tutorials/soc/compute-parameters.webp "Compute parameters")
+
+Save and exit job designer, now hover over the job, and click the run button to
+submit job.
+
+
+## 4. Results
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/dft/electronic/soc-qe.md:81]: let's add a screenshot of the result - ie. band structure, etc.

**Pull request merged**